src/suites/cts/copyImageBitmapToTexture.spec.ts

```diff
@@ -0,0 +1,83 @@
+export const description = `
+copy imageBitmap To texture tests.
+`;
+
+import { TestGroup } from '../../framework/index.js';
+
+import { GPUTest } from './gpu_test.js';
+
+export const g = new TestGroup(GPUTest);
+
+function calculateRowPitchForImageBitmap(width: number, bytesPerPixel: number): number {
+  const bytesPerRow = width * bytesPerPixel;
+  if (bytesPerRow % 256 === 0) {
+    return bytesPerRow;
+  }
+  return bytesPerRow + 256 - (bytesPerRow % 256);
```

Comment:
If bytesPerRow is 256, this returns 512, but it should return 256.
Could use `Math.ceil(bytesPerRow / 256) * 256)` or `(((bytesPerRow - 1) >> 8 + 1) << 8`

Reply:
No, if bytesPerRow is 256. it will return in the if block. so it is still 256. But I think your suggestion is better.

Reply:
Oops, you're right. I missed that.

Reply:
Done